lib/engine/share_bundle.rb

```diff
@@ -50,7 +50,8 @@ def price
     def can_dump?(entity)
       return true unless presidents_share
 
-      (corporation.player_share_holders.reject { |k, _| k == entity }.values.max || 0) >= presidents_share.percent
+      sh = corporation.player_share_holders(corporate: true)
```

Comment:
this is already default true

Reply:
Good catch. Its should not be the default, that's why.